CONTRIBUTORS.md

```diff
@@ -0,0 +1,5 @@
+# Contributors
+
+| GitHub username | Discord username | Contributions | Additional info |
+|-----------------|------------------|---------------|-----------------|
+| [mehta-avi](https://github.com/mehta-avi) | rbwings | Documentation |[My website](avimehta.info) |
```

Comment:
Also contributions should have more details

Reply:
Understood. Went back and added details on what I have been involved with so far